packages/analytics-js/src/services/ErrorHandler/utils.ts

```diff
@@ -206,6 +209,44 @@ const getErrorDeliveryPayload = (payload: ErrorEventPayload, state: ApplicationS
   return stringifyWithoutCircular<MetricServicePayload>(data) as string;
 };
 
+/**
+ * A function to get the grouping hash value to be used for the error event.
+ * Grouping hash is suppressed for non-cdn installs.
+ * If the grouping hash is an error instance, the normalized error message is used as the grouping hash.
+ * If the grouping hash is an empty string or not specified, the default grouping hash is used.
+ * If the grouping hash is a string, it is used as is.
+ * @param curErrGroupingHash The grouping hash value part of the error event
+ * @param defaultGroupingHash The default grouping hash value. It is the error message.
+ * @param state The application state
+ * @param logger The logger instance
+ * @returns The final grouping hash value to be used for the error event
+ */
+const getErrorGroupingHash = (
+  curErrGroupingHash: undefined | string | SDKError,
+  defaultGroupingHash: string,
+  state: ApplicationState,
+  logger: ILogger,
+) => {
+  let normalizedGroupingHash: string | undefined;
+  if (state.context.app.value.installType !== 'cdn') {
+    return normalizedGroupingHash;
+  }
+
+  if (!isString(curErrGroupingHash)) {
+    const normalizedErrorInstance = normalizeError(curErrGroupingHash, logger);
```

Comment:
In case of undefined `curErrGroupingHash` `normalizeError` will log some warning which is not correct.

Reply:
If the error instance is not a valid error, it would validated with a warning without even reaching this step.